### PR TITLE
John pass through

### PR DESCRIPTION
**Pull request by jagprog5**
please see #142 first

### CODE REVIEW

File: reports/final/report.tex [hunk 1]
```diff
@@ -57,16 +57,16 @@
 \pagenumbering{gobble}
 \section*{Executive Summary}
 % ERIC OWNS THIS SECTION
-The Covid-19 pandemic abruptly forced many educational institutions towards online
```
Comment (thetimmorland) [inline reports/final/report.tex:60]: Did you mean to edit the exec summary or do we need to rebase?

**Review by thetimmorland — APPROVED** [review]

File: reports/final/report.tex (hunk 1 above)
Comment (jagprog5) [reply reports/final/report.tex:60]: these edits were intentional

**Comment by EYangster** [comment]
It is showing changes from your previous branch
Can you rebase

**Comment by jagprog5** [comment]
@EYangster merged main, thanks

File: reports/final/report.tex [hunk 2]
```diff
@@ -57,16 +57,16 @@
...
-to students. Much of the onsite equipment is ill-suited for online learning due to
-its high cost and poor portability. Additionally, the take-home equipment provided
-to students during the pandemic was typically of lower quality in comparison. In
-particular, the oscilloscopes provided by the Electronics II course taught at Queen's
-University were expensive and of poor quality. They had tiny 3-inch displays and a
-small maximum bandwidth of 100 kHz. Moreover, they were single channel, limiting the
-student's ability to see how signals in a circuit are related to one another.
+The Covid-19 pandemic abruptly forced educational institutions towards online
+learning. As a result, many of these institutions were unequipped to deal with
+this new learning model. Much of the onsite equipment is ill-suited for online
```
Comment (EYangster) [inline reports/final/report.tex]: Would be better to at least mention labs, it is unclear what type of "equipment" is referred to here
E.g. Much of the onsite lab equipment ...

File: reports/final/report.tex [hunk 3]
```diff
@@ -225,17 +225,16 @@ \subsection{Hardware} % Ethan
...
-signal waveforms. Moreover, it is common with existing commercial oscilloscopes
-to have an FPGA or ASIC fulfill these real time requirements. One compromise
-that was made in the design was the use of an FPGA module rather than installing
-the chip to the PCB directly. While such a module could not be used in TeachEE
-in a mass-production setting, it greatly shortened the PCB design time, and
-allows the FPGA to be easily replaced in the event of hardware failure.
-Moreover, the FPGA combines all the sample streams and provides a single point
-of contact to the USB interface.
+All the data acquisition channels are connected to an FPGA rather than a
+microprocessor. This was decided to ensure that there is no real-time data
```
Comment (EYangster) [inline reports/final/report.tex]: I think "This decision was made to" sounds better than "This was decided to"

File: reports/final/report.tex [hunk 4]
```diff
@@ -225,17 +225,16 @@ \subsection{Hardware} % Ethan
...
+processing limitations, and that there is precise timing in our sampling of the
+signal waveforms. Moreover, an FPGA or ASCI is commonly used in existing
```
Comment (EYangster) [inline reports/final/report.tex]: Why was ASIC changed to ASCI

File: reports/final/report.tex (hunk 2 above)
Comment (jagprog5) [reply reports/final/report.tex]: I could be more specific:

> Much of the onsite equipment, such as oscilloscopes and signal generators, are ill-suited for online...

File: reports/final/report.tex (hunk 4 above)
Comment (jagprog5) [reply reports/final/report.tex]: 75f3e6314668860814d5fb514a4340b47f3ea9b2

File: reports/final/report.tex (hunk 2 above)
Comment (EYangster) [reply reports/final/report.tex]: Sure

File: reports/final/report.tex (hunk 3 above)
Comment (jagprog5) [reply reports/final/report.tex]: 6351a86831140cdb8716b9f97d305a515d1f5409

File: reports/final/report.tex (hunk 2 above)
Comment (jagprog5) [reply reports/final/report.tex]: 4ea7142ff22429b6e97ec58dfaae83ca7a177618

File: reports/final/report.tex [hunk 5]
```diff
@@ -57,16 +57,17 @@
...
-its high cost and poor portability. Additionally, the take-home equipment provided
-to students during the pandemic was typically of lower quality in comparison. In
-particular, the oscilloscopes provided by the Electronics II course taught at Queen's
-University were expensive and of poor quality. They had tiny 3-inch displays and a
-small maximum bandwidth of 100 kHz. Moreover, they were single channel, limiting the
-student's ability to see how signals in a circuit are related to one another.
+The Covid-19 pandemic abruptly forced educational institutions towards online
+learning. As a result, many of these institutions were unequipped to deal with
+this new learning model. Much of the onsite equipment, such as oscilloscopes and
+signal generators, are ill-suited for online learning due to its high cost and
```
Comment (EYangster) [inline reports/final/report.tex:63]: Equipment is singular, not plural
> , is ill-suited...

Comment (EYangster) [reply reports/final/report.tex:63]: I'll change it later

**Pull request merged**